### PR TITLE
br: docs on restore disk protection

### PR DESCRIPTION
**Pull request by BornChanger**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by YuJuncen — APPROVED** [review]

File: br/br-snapshot-guide.md [hunk 1]
```diff
@@ -68,6 +68,7 @@ tiup br validate decode --field="end-version" \
 > - 从 BR v7.6.0 开始，为了解决大规模 Region 场景下可能出现的恢复瓶颈问题，BR 支持通过粗粒度打散 Region 的算法加速恢复（实验特性）。可以通过指定命令行参数 `--granularity="coarse-grained"` 来启用此功能。
 > - 从 BR v8.0.0 开始，通过粗粒度打散 Region 算法进行快照恢复的功能正式 GA，并默认启用。通过采用粗粒度打散 Region 算法、批量创建库表、降低 SST 文件下载和 Ingest 操作之间的相互影响、加速表统计信息恢复等改进措施，快照恢复的速度有大幅提升。在实际案例中，快照恢复的 SST 文件下载速度最高提升约 10 倍，单个 TiKV 节点的数据恢复速度稳定在 1.2 GiB/s，端到端的恢复速度大约提升 1.5 到 3 倍，并且能够在 1 小时内完成对 100 TiB 数据的恢复。
 > - 从 BR v8.2.0 开始，命令行参数 `--granularity` 被废弃，粗粒度打散 Region 算法默认启用。
+> - 从 BR v8.3.0 开始，快照恢复增加了对 TiKV 以及 TiFlash 磁盘可用空间大小的检查，包括在开始阶段根据需要恢复的 sst 文件大小检查 TiKV 以及 TiFlash 磁盘可用空间大小，同时也增加了在 TiKV （v8.3.0及以上）侧下载每个 sst 文件前对磁盘剩余空间的检查。如果可用空间检查失败，恢复任务会报错失败。你可以通过设置 `--check-requirements=false` 来选择跳过恢复任务开始阶段的检查，但 TiKV 侧下载文件前的磁盘空间检查不可以跳过。
```
Comment (qiancai) [inline br/br-snapshot-guide.md:71]:
```suggestion
> - 从 BR v8.3.0 开始，快照恢复增加了对 TiKV 和 TiFlash 的磁盘可用空间检查，包括在开始阶段会根据需要恢复的 SST 文件大小检查 TiKV 和 TiFlash 是否有足够的磁盘空间，在 TiKV（v8.3.0 或更高的版本）下载每个 SST 文件前会检查 TiKV 是否有足够的磁盘空间。如果可用空间检查失败，恢复任务会报错失败。你可以通过设置 `--check-requirements=false` 来选择跳过恢复任务开始阶段的检查，但 TiKV 在下载 SST 文件前的检查无法跳过。
```

File: br/br-snapshot-guide.md [hunk 2]
```diff
@@ -68,6 +68,7 @@ tiup br validate decode --field="end-version" \
 > - 从 BR v7.6.0 开始，为了解决大规模 Region 场景下可能出现的恢复瓶颈问题，BR 支持通过粗粒度打散 Region 的算法加速恢复（实验特性）。可以通过指定命令行参数 `--granularity="coarse-grained"` 来启用此功能。
 > - 从 BR v8.0.0 开始，通过粗粒度打散 Region 算法进行快照恢复的功能正式 GA，并默认启用。通过采用粗粒度打散 Region 算法、批量创建库表、降低 SST 文件下载和 Ingest 操作之间的相互影响、加速表统计信息恢复等改进措施，快照恢复的速度有大幅提升。在实际案例中，快照恢复的 SST 文件下载速度最高提升约 10 倍，单个 TiKV 节点的数据恢复速度稳定在 1.2 GiB/s，端到端的恢复速度大约提升 1.5 到 3 倍，并且能够在 1 小时内完成对 100 TiB 数据的恢复。
 > - 从 BR v8.2.0 开始，命令行参数 `--granularity` 被废弃，粗粒度打散 Region 算法默认启用。
+> - 从 BR v8.3.0 开始，快照恢复增加了对 TiKV 和 TiFlash 的可用磁盘空间检查，包括在开始阶段会根据需要恢复的 SST 文件大小检查 TiKV 和 TiFlash 是否有足够的磁盘空间，在 TiKV（v8.3.0 或更高的版本）下载每个 SST 文件前会检查 TiKV 是否有足够的磁盘空间。如果检测到空间不足，恢复任务会报错失败。你可以通过设置 `--check-requirements=false` 来选择跳过恢复任务开始阶段的检查，但 TiKV 在下载 SST 文件前的磁盘空间检查无法跳过。
```
Comment (qiancai) [inline br/br-snapshot-guide.md:71]:
```suggestion
> - 从 BR v8.3.0 开始，快照恢复增加了对 TiKV 和 TiFlash 的可用磁盘空间检查，包括在开始阶段会根据需要恢复的 SST 文件大小检查 TiKV 和 TiFlash 是否有足够的磁盘空间，以及在 TiKV（v8.3.0 或更高的版本）下载每个 SST 文件前会检查 TiKV 是否有足够的磁盘空间。如果检测到空间不足，恢复任务会报错失败。你可以通过设置 `--check-requirements=false` 来选择跳过恢复任务开始阶段的检查，但 TiKV 在下载 SST 文件前的磁盘空间检查无法跳过。
```

File: br/br-snapshot-guide.md [hunk 3]
```diff
@@ -68,6 +68,7 @@ tiup br validate decode --field="end-version" \
 > - 从 BR v7.6.0 开始，为了解决大规模 Region 场景下可能出现的恢复瓶颈问题，BR 支持通过粗粒度打散 Region 的算法加速恢复（实验特性）。可以通过指定命令行参数 `--granularity="coarse-grained"` 来启用此功能。
 > - 从 BR v8.0.0 开始，通过粗粒度打散 Region 算法进行快照恢复的功能正式 GA，并默认启用。通过采用粗粒度打散 Region 算法、批量创建库表、降低 SST 文件下载和 Ingest 操作之间的相互影响、加速表统计信息恢复等改进措施，快照恢复的速度有大幅提升。在实际案例中，快照恢复的 SST 文件下载速度最高提升约 10 倍，单个 TiKV 节点的数据恢复速度稳定在 1.2 GiB/s，端到端的恢复速度大约提升 1.5 到 3 倍，并且能够在 1 小时内完成对 100 TiB 数据的恢复。
 > - 从 BR v8.2.0 开始，命令行参数 `--granularity` 被废弃，粗粒度打散 Region 算法默认启用。
+> - 从 BR v8.3.0 开始，快照恢复增加了对 TiKV 和 TiFlash 的可用磁盘空间检查，包括在开始阶段会根据需要恢复的 SST 文件大小检查 TiKV 和 TiFlash 是否有足够的磁盘空间，以及在 TiKV（v8.3.0 或更高的版本）下载每个 SST 文件前会检查 TiKV 是否有足够的磁盘空间。如果检测到空间不足，恢复任务会报错失败。你可以通过设置 `--check-requirements=false` 来选择跳过恢复任务开始阶段的检查，但 TiKV 在下载 SST 文件前的磁盘空间检查无法跳过。
```
Comment (Oreoxmt) [inline br/br-snapshot-guide.md:71]:
```suggestion
> - 从 BR v8.3.0 开始，快照恢复增加了对 TiKV 和 TiFlash 可用磁盘空间的检查，包括在开始阶段会根据需要恢复的 SST 文件大小检查 TiKV 和 TiFlash 是否有足够的磁盘空间，以及在 TiKV（v8.3.0 或之后的版本）下载每个 SST 文件前会检查 TiKV 是否有足够的磁盘空间。如果任一阶段检测到可用空间不足，恢复任务会报错失败。你可以通过设置 `--check-requirements=false` 来选择跳过恢复任务开始阶段的检查，但 TiKV 在下载 SST 文件前的磁盘空间检查无法跳过。
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Review by qiancai — APPROVED** [review]

**Pull request merged**